lib/osutil/drives_windows.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2016 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// +build windows
+
+package osutil
+
+import (
+	"bytes"
+	"syscall"
+	"unsafe"
+)
+
+func GetDriveLetters() []string {
+	drives := []string{}
+
+	kernel32, err := syscall.LoadDLL("kernel32.dll")
+	if err != nil {
+		return drives
+	}
```

Comment:
It matters only in that when looking at this I need to keep track of if we've actually done anything with `drives` to see what we return in the error cases. That's trivial in the first case but when looking at line 34 I actually have to parse lines 18-33 as well to understand what it means.

As an aside, shouldn't we just return `([]string, error)` and let the API return a 500 when this fails on Windows? The empty success response isn't meaningful by itself, is it?